addons/mail/models/mail_thread.py

```diff
@@ -3142,3 +3142,7 @@ def _get_mail_redirect_suggested_company(self):
         if 'company_id' in self:
             return self.company_id
         return False
+
+    @api.model
+    def _get_attachment_domain(self, thread, data):
```

Comment:
I don't think thread mixin should have a method just for a given domain, that is tweaked for some strange reason in documents. This should be done as a tool function in controllers directly, this is not a business method :) .

Reply:
@tde-banana-odoo  Done.Thanks